css/properties.json

```diff
@@ -3240,6 +3240,26 @@
     ],
     "status": "standard"
   },
+  "font-variation-settings": {
+    "syntax": "normal | [ &lt;string&gt; &lt;number&gt; ] #",
+    "media": "visual",
+    "inherited": true,
+    "animationType": "discrete",
```

Comment:
The [spec. says](https://drafts.csswg.org/css-fonts-4/#low-level-font-variation-settings-control-the-font-variation-settings-property):

> Animating font-variation-settings is possible using a mechanism similar to animating the transform property: Two declarations of font-feature-settings can be animated between if they are "like".

So, `discrete` is wrong here. There needs to be a short description added to l10n/css.json explaining the animation type and a referencing keyword here.

Sebastian